modules/web-console/frontend/views/templates/stacktrace-viewer.tpl.pug

```diff
@@ -0,0 +1,30 @@
+//-
```

Comment:
Please move this file under queries module. We should not create new global route/dialog templates, it's a legacy way of doing things. In perfect scenario, this would be a component template.

Reply:
Moved as template of separated component.